### ISSUE TITLE
Can not Focus TexField

### ISSUE THREAD

**Issue by NicolasCELLA** [Dn-a/flutter_inner_drawer#30]
Hello ! 

I have a probleme, i can't focus a TextField after used Navigator.push() 

I have an example :

https://www.youtube.com/watch?v=ppQUZIc8e7s&feature=youtu.be

```
InnerDrawer(
   scaffold: Stack(
        children: <Widget>[
          TabBarView(
            physics: NeverScrollableScrollPhysics(),
            controller: _pageController,
            children: <Widget>[
              MainKomitScreen(),
              MainSearchScreen(), // TEXTFIELD INSIDE THIS WIDGET
              MainInboxScreen(),
            ]
          ),
    ),
)
```

if you have an idea why 😇

**Comment by NicolasCELLA** [comment]
update:

the Problem is from FocusScope

```
child: RepaintBoundary(
                          child: FocusScope(
                            node: _focusScopeNode,
                            child: _scaffold()),
                        )),
```

if i remove FocusScope i dont have the problem !

**Comment by Dn-a** [comment]
hi @NicolasCELLA, have you tried to follow this example?

**Comment by NicolasCELLA** [comment]
Yes of course ! But if you look the end of the video ! I can't tab inside the textField ! 
When a don't use your Widget this working and when inside your Widget i remove 
```
FocusScope(
                            node: _focusScopeNode,
```
This working !  I dont know why but maybe in your widget something is not unfocused !

**Comment by Dn-a** [comment]
@NicolasCELLA I removed focusScope, at the moment it seems to work properly even without. try it and let me know.

**Comment by NicolasCELLA** [comment]
Yes this work very well ! But why focusScope why used ? It's was important ? Because a don't see difference without

**Comment by Dn-a** [comment]
@NicolasCELLA it is used to move the focus between one widget and another. It didn't work because I forgot to enter a parameter `node: _focusScopeNode`.
Among other things, I had to put it as a wrapper for the child widgets and not for the scaffold.
Try version 0.5.5 + 2 and let me know

**Comment by NicolasCELLA** [comment]
This version work fine 👌 ! But i have a recommandation for your widget :
The possibility of unfocus or not !

Actually :  
https://www.youtube.com/watch?v=ynb7kHHsdNI

without :
```
FocusScope(node: _focusScopeNode, child: _animatedChild()),
```
but with only :
```
 _animatedChild()
```
result:
https://www.youtube.com/watch?v=qKtGlzeNw2M

**Comment by Dn-a** [comment]
@NicolasCELLA could you show the source code that creates this problem? I have made several tests and it behaves correctly
